### PR TITLE
Always remove testexp.tar test file

### PR DESCRIPTION
**Pull request by icecrime**
- File was not removed in case of failure
- File was removed from `/tmp` but really created in `integration-cli/`

### CODE REVIEW

**Comment by duglin** [comment]
LGTM

File: integration-cli/docker_cli_export_import_test.go [hunk 1]
```diff
@@ -91,7 +93,5 @@ func TestExportContainerWithOutputAndImportImage(t *testing.T) {
 	deleteContainer(cleanedContainerID)
 	deleteImages("repo/testexp:v1")
```
Comment (duglin) [inline integration-cli/docker_cli_export_import_test.go]: while in there, perhaps make these into defer's too ?

**Comment by icecrime** [comment]
Thanks @duglin, updated!

**Comment by duglin** [comment]
assuming janky is happy, still LGTM

**Comment by duglin** [comment]
@icecrime minor errors

**Comment by estesp** [comment]
@icecrime no compile :( your defer that has `cleanedContainerID` has to be after the assignment, right?

**Comment by icecrime** [comment]
Oh god, I'm sorry :-/ Wanted to do this fast, and ended up doing :hankey:

**Comment by duglin** [comment]
squash?

**Comment by crosbymichael** [comment]
LGTM

**Comment by duglin** [comment]
@icecrime can you squash this then LGTM

**Comment by icecrime** [comment]
Done!

**Comment by HuKeping** [comment]
CI may fail as #12467

**Comment by duglin** [comment]
janky errors are not related to this so ignoring....

**Pull request merged**